blocks/showcase/index.js

```diff
@@ -0,0 +1,132 @@
+import './style.scss';
+import './editor.scss';
+
+const { __ } = wp.i18n;
+
+const {
+	registerBlockType,
+} = wp.blocks;
+
+registerBlockType( 'rtgb/showcase', {
+
+	title: __( 'Showcase' ),
+	icon: 'search',
+	category: 'layout',
+	description: __( 'Use for showcase' ),
+
+	attributes: {
+
+		showCaseTitle: {
+			type: 'array',
+			field: {
+				type: 'rich-text',
+				className: 'showcase-title',
+				placeholder: __( 'Showcase Title' ),
+				tagName: 'h3',
+			},
+			selector: '.showcase-title',
+			source: 'children',
+		},
+
+		showCaseContent: {
+			type: 'array',
+			field: {
+				type: 'rich-text',
+				className: 'showcase-content',
+				placeholder: __( 'Showcase description' ),
+				tagName: 'div',
+				multiline: 'p',
+			},
+			selector: '.showcase-content',
+			source: 'children',
+		},
+
+		showCaseImage: {
+			type: 'object',
+			field: {
+				type: 'image',
+				buttonText: __( 'Upload' ),
+				imagePlaceholder: true,
+				removeButtonText: __( 'Remove' ),
+			},
+		},
+
+		showCaseLink: {
+			type: 'string',
+			field: {
+				type: 'link',
+				placement: __( 'inspector' ),
+				label: __( 'Showcase link' ),
+			},
+		},
+	},
+
+	getEditWrapperProps() {
+		return { 'data-align': 'wide' };
+	},
+
+	edit( props, middleware ) {
+		const {
+			attributes: {
+				showCaseLink,	
+			},
+		} = props;
+
+		const className = props.className ? props.className : '';
+
+		return (
+			<div className={ className + ' showcase-wrapper alignwide' }>
+				<div className="image-container">
+					{ middleware.fields.showCaseImage }
+				</div>
+
+				<div className="info-container">
+					{ middleware.inspectorControls }
+					{ middleware.fields.showCaseTitle }
+					{ middleware.fields.showCaseContent }
+					{ middleware.fields.showCaseLink }
+					{ showCaseLink ? <a href={ showCaseLink } title="__( 'Read More' )" className="button secondary">{ __( 'Read More' ) }</a> : '' }
```

Comment:
`"__( 'Read More' )"` is a string not JS

Reply:
Fixed.